mct/m_SPMDutils.F90

```diff
@@ -24,6 +24,16 @@
 !
 ! !INTERFACE:
 
+! Code added as a work around for poor performance or incorrect functionality
+! in MPI libraries
+!
+#ifndef _USE_MPI_RSEND
```

Comment:
Edit the CPPDEFS in the Macros file after "case.setup" has been run.  That will pass it to all files but that should be harmless.